app/controllers/tickets_controller.rb

```diff
@@ -59,12 +59,13 @@ def show
   def index
     @agents = User.agents
 
-    params[:status] ||= 'open'
+    params[:status] ||= 'open' unless params[:user_id]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.